Adding the ability to specify PolicyDocument in aws.iam.Policy (#601)
pulumi/pulumi-aws: CHANGELOG.md

* Add constants for Route53 record types
* Subscription resources will now be parented by default by the resource they were created off of.
  This has been implemented using 'aliases' so this will not have any effect on existing stacks.

> * Adding the ability to specify `PolicyDocument` type as the `policy` field in `aws.iam.Policy`

Reviewer: ```suggestion
* Add the ability to pass a `PolicyDocument` in the `policy` field in `aws.iam.Policy`
```

Author: Will follow this up on the subsequent PR for the next enhancement
